src/test/java/com/logigear/LoginTest.java

```diff
@@ -14,6 +14,21 @@
     private final HomePage homePage = new HomePage();
     private final LoginPage loginPage = new LoginPage();
 
+    @Description("TC02 - User can't login with blank Username textbox")
+    @Test
```

Comment:
Use the wrong annotation. You should use this:
@test(description="...")

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.